src/Sylius/Component/Promotion/Model/ActionInterface.php

```diff
@@ -19,6 +19,7 @@
 interface ActionInterface extends ResourceInterface
 {
     const TYPE_FIXED_DISCOUNT = 'fixed_discount';
+    const TYPE_ITEM_PERCENTAGE_DISCOUNT = 'item_percentage_discount';
     const TYPE_PERCENTAGE_DISCOUNT = 'percentage_discount';
```

Comment:
`shipping_percentage_discount` is missing + we should think about adding prefix for order discount `order_percentage_discount`?